samples/Samples.h

```diff
@@ -15,12 +15,18 @@ extern "C" {
 #define NUMBER_OF_H264_FRAME_FILES               1500
 #define NUMBER_OF_OPUS_FRAME_FILES               618
 #define DEFAULT_FPS_VALUE                        25
+#define DEFAULT_AUDIO_CHANNELS                   2
+#define DEFAULT_AUDIO_SAMPLE_RATE                48000
+#define DEFAULT_AUDIO_BITS_PER_SAMPLE            16
 #define DEFAULT_MAX_CONCURRENT_STREAMING_SESSION 10
+#define DEFAULT_HEIGHT                           720
+#define DEFAULT_WIDTH                            1280
```

Comment:
Adding units will help - may be self explanatory to most.
DEFAULT_AUDIO_SAMPLE_RATE_HZ
DEFAULT_VIDEO_HEIGHT_PIXELS

Reply:
Done!